docs/05-awesomerc.md.lua

```diff
@@ -242,7 +242,16 @@ local rc = assert(io.open(rcfile))
 
 local doc_block = false
 
-local output = {}
+local output, output_script = {}, {[[
+---------------------------------------------------------------------------
+--- The default rc.lua file.
+--
+-- A copy is this file is usually installed in `/etc/xdg/awesome/`.
+--
+-- See [The declarative layout system](../documentation/05-awesomerc.md.html)
+-- for a more documented version of this file.
```

Comment:
s/for a more documented version/for a version with additionally (auto-generated) comments/